controllers/controller.js

```diff
@@ -73,11 +76,24 @@ function addComments(req, res, next) {
   })
 }
 
+function getUpdatedArticles(req, res, next) {
+  const { article_id } = req.params
+
```

Comment:
For readability, it's best to consistently destructure, or use longhand
You're using both syntaxes in this function, destructuing `article_id` and using longhand for `body`